en/orm/saving-data.rst

```diff
@@ -740,6 +740,20 @@ persisted::
 
 Without the call to ``dirty()`` the updated tags will not be saved.
 
+When saving belongsToMany Associations, it can be relevant to save some additional data to the Joint Table.
```

Comment:
Could you add a short example first of using `link()`? It would be nice for people to understand that exists before jumping to saving additional data in to joint table